pallets/loans/Cargo.toml

```
@@ -34,18 +34,18 @@ pallet-balances = { git = "https://github.com/paritytech/substrate",  default-fe
 pallet-uniques = { git = "https://github.com/paritytech/substrate",  default-features = false , optional = true, branch = "polkadot-v0.9.24" }
 pallet-pools = { path = "../pools", optional = true, default-features = false}
 
-orml-tokens = { git = "https://github.com/NunoAlexandre/open-runtime-module-library", default-features = false, optional = true, branch = "polkadot-v0.9.24" }
+orml-tokens = { git = "https://github.com//open-web3-stack/open-runtime-module-library", default-features = false, optional = true, branch = "polkadot-v0.9.24" }
```

Looks like you introduced additional `/`s (not just in this instance):
```suggestion
orml-tokens = { git = "https://github.com/open-web3-stack/open-runtime-module-library", default-features = false, optional = true, branch = "polkadot-v0.9.24" }
```

---

Thanks for the catch Jeroen, fixed!